src/main/java/nextstep/courses/domain/SessionImage/SessionImage.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain.SessionImage;
+
+public class SessionImage {
+
+    private final ImageCapacity imageCapacity;
+    private final ImageType imageType;
+    private final ImageSize imageSize;
```

Comment:
```suggestion
    private final ImageCapacity capacity;
    private final ImageType type;
    private final ImageSize size;
```
이 부분도 위와 같은 내용입니다! 😄

Reply:
넵 동일하게 수정 완료했습니다~!